crates/ruff_linter/src/checkers/ast/mod.rs

```diff
@@ -530,6 +530,26 @@ impl<'a> Checker<'a> {
         f(&mut checker, self);
         self.semantic_checker = checker;
     }
+
+    /// Attempt to create an [`Edit`] that imports `member`.
+    ///
+    /// On Python <`python_version`, `member` is imported from `typing_extensions`, while on Python
+    /// >=`python_version`, it is imported from `typing`.
+    pub(crate) fn import_from_typing(
+        &self,
+        member: &str,
+        position: TextSize,
+        python_version: PythonVersion,
```

Comment:
nit: maybe this parameter name would be slightly clearer about what the parameter indicates?

```suggestion
        version_added_to_typing: PythonVersion,
```

Reply:
I like that, thanks!